vllm/model_executor/layers/quantization/gguf.py

```diff
@@ -0,0 +1,125 @@
+from typing import Any, Dict, List, Optional
+
+import torch
+from gguf.constants import GGML_QUANT_SIZES
+from torch.nn.parameter import Parameter, UninitializedParameter
+
+from vllm import _custom_ops as ops
+from vllm.model_executor.layers.linear import LinearBase, LinearMethodBase
+from vllm.model_executor.layers.quantization.base_config import (
+    QuantizationConfig)
+from vllm.model_executor.layers.vocab_parallel_embedding import (
+    VocabParallelEmbedding)
+from vllm.model_executor.utils import set_weight_attrs
+
+
+class GGUFConfig(QuantizationConfig):
+    """Config class for GGUF."""
+
+    def __init__(self, ) -> None:
+        pass
+
+    def __repr__(self) -> str:
+        return ("GGUFConfig()")
+
+    @property
+    def merge_weight(self) -> bool:
+        return False
+
+    @property
+    def is_neox_style(self) -> bool:
+        return False
```

Comment:
Why is this defined by the quant method? This should be a function of the model definition

Reply:
OK, we will remove this